src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -2945,6 +2963,13 @@ private BuildingRequestEventArgs FireBuildingRequest(BuildingRequestEventArgs bu
             {
                 this.InnerBuildingRequest(this, buildingRequestEventArgs);
 
+                string contentType = buildingRequestEventArgs.HeaderCollection.GetHeader(ContentType);
+                
+                if(!string.IsNullOrEmpty(contentType) && contentType.Contains(MimeIeee754CompatibleHeaderTrue))

```

Comment:
This performs a case sensitive search for the IEEE754 parameter... from limited [research here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types) it is the value part's case which may or may not have specific meaning.

Reply:
noted and fixed.